back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/user_digest.rb

```diff
@@ -114,14 +114,17 @@ def notifications_counts
     end
 
     def top_ideas
+      trending_ideas.limit N_TOP_IDEAS
+    end
+
+    def trending_ideas
       ti_service = TrendingIdeaService.new
-      top_ideas = IdeaPolicy::Scope.new(nil, Idea).resolve
+      ideas = IdeaPolicy::Scope.new(nil, Idea).resolve
         .published
         .includes(:comments)
 
-      truly_trending_ids = ti_service.filter_trending(top_ideas).ids
-      top_ideas = ti_service.sort_trending top_ideas.where(id: truly_trending_ids)
-      top_ideas.limit N_TOP_IDEAS
+      trending_ids = ti_service.filter_trending(ideas).ids
+      ti_service.sort_trending ideas.where(id: trending_ids)
```

Comment:
Refactored so that `content_worth_sending?` and `top_ideas` more clearly share code, and therefore have very similar logic.